docs/astro.config.mjs

```diff
@@ -8,6 +8,7 @@ export const locales = {
   ja: { label: '日本語', lang: 'ja' },
   fr: { label: 'Français', lang: 'fr' },
   it: { label: 'Italiano', lang: 'it' },
+  zh: { label: '简体中文', lang: 'zh' },
```

Comment:
Happy to add this in a future PR, but given this PR doesn’t add any docs translations yet, I’d suggest we leave Chinese out of the Starlight docs site menu for now — your changes to the core package will still work for other Starlight users.

```suggestion
```